docs/installation/mac.md

```diff
@@ -89,6 +94,95 @@ To start elasticsearch as a daemonized process:
 ./bin/elasticsearch -d
 ```
 
+### Installing Elasticsearch with Homebrew
+
+The following directions were
+[taken from the Elasticsearch docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/brew.html).
+
+```shell
+brew tap elastic/tap
+brew install elastic/tap/elasticsearch-oss
+```
+
+After installation you can manually test if the Elasticsearch server starts by
+issuing the command `elasticsearch` in the shell. You can then start the server
+as a service with `brew services start elastic/tap/elasticsearch-oss`.
+
+You can find further info on your local ES installation by typing
+`brew info elastic/tap/elasticsearch-oss`.
+
+#### Troubleshooting startup issues
```

Comment:
are these issues specific to installing with homebrew? if so, we should nest them under "installing with homebrew? to make that clear.

Reply:
Great catch, they were in the beginning, then I moved around the section and made a mess :D Let me fix it

Reply:
@vaidehijoshi I they already are, it's just `git diff` playing games: https://github.com/thepracticaldev/dev.to/blob/947ffcf1d166d8df74929e3970dfddad2215ad58/docs/installation/mac.md

Reply:
Oh great, I missed that in the diff. 👍